webtester-core/src/test/java/integration/AbstractWebTesterIntegrationTest.java

```diff
@@ -54,6 +55,10 @@ protected String getHTMLFilePath() {
 
     // UTILITY
 
+    protected void assertPageObjectCanBeInitialized(PageObject object) {
```

Comment:
Thanks!
I'll keep this commit out, but accept the rest of your pull request. You can add the commit to a future PR where the test method is used.